### PR TITLE
Features/ordina architecture kata post

### PR DESCRIPTION
**Pull request by kencoenen**

### CODE REVIEW

File: _posts/2017-10-18-Ordina-Architecture-Kata.md [hunk 1]
```diff
@@ -160,8 +161,14 @@ Our kata for today — AM.I.SCK
  - Enable parts of the system for direct patient usage
  - Conversations are not considered medical records
 
- The sixteen attendees were divided in groups of four.
- We had to draw the different architectural views for the AM.I.SCK platform.
+The sixteen attendees were divided in groups of four.
+We had to draw the different architectural views for the AM.I.SCK platform.
```
Comment (bart-blommaerts) [inline _posts/2017-10-18-Ordina-Architecture-Kata.md]: We had to create ...

I emphasised the drawing too much during the theory. It's not only about diagraming.

File: _posts/2017-10-18-Ordina-Architecture-Kata.md [hunk 2]
```diff
@@ -160,8 +161,14 @@ Our kata for today — AM.I.SCK
...
+
+Each team had fifteen to twenty minutes to brainstorm about the case and draw the first four views.
```
Comment (bart-blommaerts) [inline _posts/2017-10-18-Ordina-Architecture-Kata.md]: and create the first four views on a whiteboard together.

File: _posts/2017-10-18-Ordina-Architecture-Kata.md (hunk 1 above)
Comment (kencoenen) [reply _posts/2017-10-18-Ordina-Architecture-Kata.md]: Sentence removed, I said the same in the next paragraph.

**Comment by kencoenen** [comment]
Feedback processed

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+In his conference talk and blog post <a href="https://ordina-jworks.github.io/architecture/2017/06/21/pragmatic-architecture-today.html" target="_blank">Pragmatic Architecture, Today</a>, Bart Blommaerts discusses the need to think about Software Architecture. 
+
+Since this is very relevant to this Architecture Kata, we recap quickly what we learned back then.
+
+### Why do we need an architecture?
+
+We need to build a system.
+A system is build for stakeholders.
+Customers, user, developers, ... are all stakeholders of a particular system.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: user**s**?

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+OODA is a re-entrant feedback loop, that consists of four stages:
+
+1. Observe: Listen to customers, gather requirements, available resources, ...
+2. Orient: Assess comparable systems, use your experience to make sense of your earlier observations.
+3. Decide: From the orientation stage, multiple alternatives might need to be considered. 
+In the decision stage, we take a decision. 
+4. Act: Act on your decision, implement.
+
+An exercise that can help you in the different stages, is to start with some bullet points and then write these out explicitely.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: Sounds better imo:
... and then writing them out **explicitly**.

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+Avoid fluffy diagrams and mixed abstractions.
+Don't mix eg. user interaction information with data flow information.
+
+### Decision log
+
+Document your decisions and alternatives in a __Decision log__, also known as Architecture Decision Record (ADR).
+It will prove itself useful in the future and requires you to think about a decision.
+
+There's no need to invent the wheel here.
+There are several templates for different use cases available on the internet, for example in <a hreaf="https://github.com/joelparkerhenderson/architecture_decision_record" target="_blank">this ADR repo</a> on Github.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: typo?
hreaf -> href

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+Bart explained the different views with sharp-cut examples.
+
+__Context View__ — Describes the relationships, dependencies and interactions between the system and its environment.
+Added in the second print of the book.
+Bart thinks this might be the most important view of them all.
+Every component is a part of the greater system.
+
+__Functional View__ — Defines the architectural elements that deliver the systems functionality.
+It documents the systems functional structure.
+You can make decisions on a functional level eg. Two components are doing similar things.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: Two -> two

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+Should they be separate components?
+
+__Information View__ — Models the system data and its state.
+The purpose of many applications today is capturing data.
+
+> Sidenote: Data modeling can be a long and complex process.
+As an architect, you need to do data modeling at an architecturally significant level of detail.
+Go to the level of detail that is needed for your team of developers.
+
+__Concurrency View__ — Describes the concurrency structure of the system, mapping functional elements to concurrency units to clearly identify the parts of the system that can execute concurrently. eg. process a file in blocks.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: Need to remove the dot in between "concurrently" and "eg."?

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+Make sure you include all information relevant for deploying the application, eg. OS, Apache HTTPD, Tomcat, etc.
+
+__Operational view__ — Describe how the system will be operated, administered, and supported when it is running in its production environment.
+You can use a state chart to describe the operations process.
+
+## Perspectives
+
+Perspectives shape the views for non-functional requirements.
+
+When you introduce perspectives, you'll have to make tradeoffs.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: Trade-offs in British English.

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+An architectural decision will favor certain perspectives and at the same time hinder other perspectives.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: favour in British English.
... and the same time, hinder other ... (comma)

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+For example, strong encryption favors security but hinders performance.
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: Favours

File: _posts/2017-10-20-Ordina-Architecture-Kata.md [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+## Links and resources
+
+- Recommended reading: 
+<a href="https://www.goodreads.com/book/show/11686849-software-systems-architecture" target="_blank">Software Systems Architecture</a> by Eoin Woods and Nick Rozanski.
+In this book, they discuss Viewpoints and Perspectives
+- <a href="http://nealford.com/katas/about.html" target="_blank">Architectural Katas</a> on neilford.com
+- <a href="https://archkatas.herokuapp.com/" target="_blank">https://archkatas.herokuapp.com/</a>
+- <a href="https://groups.google.com/forum/#!forum/architecturalkatas" target="_blank">ArchitecturalKatas</a> Google User Group
+- <a href="http://liminalthinking.com/" target="_blank">Liminal Thinking</a>
+- <a href="http://www.enterpriseintegrationpatterns.com/">Enterprise Integration Patterns</a>
```
Comment (yannickdeturck) [inline _posts/2017-10-20-Ordina-Architecture-Kata.md]: Missing the target blank thing.

**Comment by kencoenen** [comment]
Feedback @yannickdeturck processed!

**Pull request merged**